src/connection.ts

```diff
@@ -210,6 +211,36 @@ export class Connection extends JSForceConnection {
     return super._baseUrl();
   }
 
+  /**
+   * Will deploy a recently validated deploy request
+   *
+   * @param options.id = the id of the deploy to quickly validate
+   * @param options.rest = a boolean whether or not to use the REST API
+   */
+  public async deployRecentValidation(options: recentValidationOptions): Promise<JsonCollection> {
+    if (options.rest) {
+      const url = `${this.instanceUrl.replace(
+        /\/$/,
+        ''
+      )}/services/data/v${this.getApiVersion()}/metadata/deployRequest`;
+      const messageBody = JSON.stringify({
+        validatedDeployRequestId: options.id,
+      });
+      const requestInfo = {
+        method: 'POST',
+        url,
+        body: messageBody,
+      };
+      const requestOptions = { headers: 'json' };
+      return this.request(requestInfo, requestOptions);
```

Comment:
This doesn't auto-refresh the auth.  Not sure it's a huge deal since people will generally run this immediately after a successful checkOnly deploy, but it would be a bug.  That could be addressed in another work item though.